.vscode/launch.json

```
@@ -0,0 +1,28 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Debug single lean file",
+            "cwd": "${workspaceFolder}/library",
+            "program": "${workspaceRoot}/build/debug/shell/lean",
```

```suggestion
            "program": "${workspaceRoot}/bin/lean",
```
Would this also work for you?  Then we wouldn't hardcode the cmake build path.

---

That's almost certainly fine; does cmake always copy the most recently-build target to that bin directory?

---

(tested, this works just fine)

---

Yes, as long as you call `make bin_lean` or `make standard_lib` or `make` (instead of `make lean`).